src/Server/RestHTTPRequestHandler.cpp

```diff
@@ -406,14 +407,21 @@ RestHTTPRequestHandler::RestHTTPRequestHandler(IServer & server_, const String &
 }
 
 /// FIXME : Get the corresponding database according to the user name
-String RestHTTPRequestHandler::getDatabaseByUser(const String & user) const
+String RestHTTPRequestHandler::getDatabaseByUser(const String & user_name) const
 {
     String database = "default";
 
+    /// FIXME : Temporarily return databse according to the configuration
+    if (auto user = server.context()->getAccessControlManager().tryRead<User>(user_name))
+    {
+        if (!user->database.empty())
+            database = user->database;
+    }
+
     if (database.empty())
```

Comment:
It seems this `if` condition was false always.

Reply:
This place is made for later